cppcache/integration-test/BuiltinCacheableWrappers.hpp

```diff
@@ -840,14 +840,14 @@ class CacheableBytesWrapper : public CacheableWrapper {
     auto randArr =
         CacheableHelper::randomArray<uint8_t>(maxSize, UCHAR_MAX);
     m_cacheableObject = CacheableBytes::create(
-        reinterpret_cast<const int8_t*>(randArr.data()), maxSize);
+        std::vector<int8_t>(randArr.data(), randArr.data() + maxSize));
```

Comment:
I wonder if since the intent is to have a random `CacheableBytes` (`int8_t`) then why not have the initial `randomArray` call return a `vector<int8_t>`?

Reply:
Yes, that would be better.  I'll give it a quick stab.